README.md

```diff
@@ -329,6 +329,7 @@ A third-party Golang SDK for Jackett is available from [webtor-io/go-jackett](ht
  * Empornium (EMP) [![(invite needed)][inviteneeded]](#)
  * eShareNet
  * eStone (XiDER, BeLoad, BigTorrent)
+ * Exitorrent
```

Comment:
```suggestion
 * Exitorrent.org
```
if the site is not related to exitorrent.eu then lets tell them apart
if it is related then leave as is.